src/ui/features/onboarding/Success/Success.tsx

```diff
@@ -94,6 +101,21 @@ export function Success() {
     };
   }, [fireConfetti]);
 
+  const { data: currentReferrer } = useQuery({
+    queryKey: ['currentReferrer'],
+    queryFn: async () => {
+      const result = await readReferrer();
```

Comment:
let's rename this function though. 
What do you think about `getSavedReferrerData`?

Reply:
Agree, reads better!